packages/devextreme/js/__internal/ui/text_box/m_text_editor.base.ts

```diff
@@ -591,6 +591,7 @@ const TextEditorBase = Editor.inherit({
       if (this.hasActionSubscription(`on${event}`)) {
         const action = this._createActionByOption(`on${event}`, { excludeValidators: ['readOnly'] });
 
+        eventsEngine.off($input, addNamespace(event.toLowerCase(), this.NAME));
```

Comment:
_refreshEvents already unsubscribes, so i suggest remove this change and use _refreshEvents() instead of _renderEvents [here](https://github.com/DevExpress/DevExtreme/blob/c62064b1f1b53d7d5bea8188176ee3a0bb83eb99/packages/devextreme/js/__internal/ui/text_box/m_text_editor.base.ts#L201)

Reply:
[889c03f](https://github.com/DevExpress/DevExtreme/pull/27660/commits/889c03f35413c3be25904a85b8640bb2a203714f)